### PR TITLE
Fixes part of #632 : RecyclerView implementation using BindableAdapter in TopicReview/Revision

### PR DESCRIPTION
**Pull request by Sarthak2601**
## Explanation
 Fixes part of #632 -- TopicReview/Revision 

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/topic/revision/revisionitemviewmodel/TopicRevisionItemViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import org.oppia.app.topic.revision.RevisionSubtopicSelector
+import org.oppia.app.topic.revision.TopicRevisionFragment
+import org.oppia.app.model.Subtopic
+
+/** [ViewModel] for child views of recycler view present in the [TopicRevisionFragment]. */
+class TopicRevisionItemViewModel(
+  val subtopic: Subtopic,
+  val onRevisionItemPressed: RevisionSubtopicSelector
+) : ViewModel()
+
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/topic/revision/revisionitemviewmodel/TopicRevisionItemViewModel.kt]: Remove this extra blank line.

**Review by rt4914 — APPROVED** [review]
Great implementation. LGTM, just one nit change suggested.

**Comment by rt4914** [comment]
@Sarthak2601 Your linter test cases are failing on circle CI. To solve this just take the latest develop code and merge it in your branch.

File: app/src/main/java/org/oppia/app/topic/revision/revisionitemviewmodel/TopicRevisionItemViewModel.kt (hunk 1 above)
Comment (Sarthak2601) [reply app/src/main/java/org/oppia/app/topic/revision/revisionitemviewmodel/TopicRevisionItemViewModel.kt]: Done:)

**Comment by Sarthak2601** [comment]
> @Sarthak2601 Your linter test cases are failing on circle CI. To solve this just take the latest develop code and merge it in your branch.

Okay 👍

**Comment by Sarthak2601** [comment]
@rt4914 The linter test cases are passing now. PTAL :)

**Pull request merged**